### ISSUE TITLE
Prevent font embedding

### ISSUE THREAD

**Issue by frontdevde** [ehynds/grunt-image-embed#17]
Hey there, 

unless I missed something there seems to be no way to exclude fonts from being embedded, too. I just want images to be embedded, a way of specifing a whitelist with which file name extensions should be embedded would be awesome!

So long,
Michael

**Comment by gobwas** [comment]
+1..

**Comment by mokkabonna** [comment]
+1

Strike the below comment, seems my less compiler was moving the comments around...

also the /_ImageEmbed:skip_/ seems to do nothing for me like this:

```
src: url('../fonts/myfontLight.eot');/*ImageEmbed:skip*/
src: url('../fonts/myfontLight.eot?#iefix') format('embedded-opentype'), url('../fonts/myfontLight.woff') format('woff'), url('../fonts/myfontLight.ttf') format('truetype'), url('../fonts/myfont.svg#myfont') format('svg');/*ImageEmbed:skip*/
```

It still includes them.

**Comment by ghost** [comment]
+1

**Comment by ghost** [comment]
This works for me
My SCSS:

```
@font-face {
  font-family: 'FontAwesome';
  src: url('#{$FontAwesomePath}/fontawesome-webfont.eot?v=#{$FontAwesomeVersion}'); /*ImageEmbed:skip*/
  src: url('#{$FontAwesomePath}/fontawesome-webfont.eot?#iefix&v=#{$FontAwesomeVersion}') format('embedded-opentype'), /*ImageEmbed:skip*/
    url('#{$FontAwesomePath}/fontawesome-webfont.woff?v=#{$FontAwesomeVersion}') format('woff'), /*ImageEmbed:skip*/
    url('#{$FontAwesomePath}/fontawesome-webfont.ttf?v=#{$FontAwesomeVersion}') format('truetype'), /*ImageEmbed:skip*/
    url('#{$FontAwesomePath}/fontawesome-webfont.svg#fontawesomeregular?v=#{$FontAwesomeVersion}') format('svg'); /*ImageEmbed:skip*/
  font-weight: normal;
  font-style: normal;
}
```

then in https://github.com/ehynds/grunt-image-embed/blob/master/tasks/lib/encode.js#L18
I have this:

```
var rImages = /([\s\S]*?)(url\(([^)]+)\))(?!.*[;,]?\s*\/\*\s*ImageEmbed:skip\s*\*\/)|([\s\S]+)/img;
```

**Comment by ehynds** [comment]
landed in #34
